en/datatypes/ref.adoc

```diff
@@ -0,0 +1,225 @@
+= Ref! datatype
+:toc:
+:numbered:
+
+== Abstract
+
+`ref!` values denote named references, such as social media handlers or munged e-mail addresses with an implied domain name.
```

Comment:
"social media handles". (typo)

Not so much munged emails, but names or IDs _within_ a domain, because the domain in emails comes _after_ the ID.